go/consensus/tendermint/full/full.go

```diff
@@ -1500,6 +1507,8 @@ func init() {
 	Flags.String(cfgCoreExternalAddress, "", "tendermint address advertised to other nodes")
 	Flags.String(CfgABCIPruneStrategy, abci.PruneDefault, "ABCI state pruning strategy")
 	Flags.Uint64(CfgABCIPruneNumKept, 3600, "ABCI state versions kept (when applicable)")
+	Flags.Bool(CfgCheckpointerDisabled, false, "Disable the ABCI state checkpointer")
+	Flags.Duration(CfgCheckpointerInterval, 1*time.Minute, "ABCI state checkpointer interval")
```

Comment:
It should be noted that this is the check interval (e.g., how often to check whether a new checkpoint needs to be taken) not the checkpoint interval.